autoemulate/experimental/calibration/bayes.py

```diff
@@ -135,22 +167,33 @@ def model(self, predict: bool = False):
                 min_val, max_val = self.parameter_range[param]
                 sampled_val = pyro.sample(param, dist.Uniform(min_val, max_val))
                 param_list.append(sampled_val.to(self.device))
+                self.logger.debug(
+                    "Sampled parameter '%s' from Uniform(%s, %s)",
+                    param,
+                    min_val,
+                    max_val,
+                )
```

Comment:
One usually does at least 1000k samples so this would probably be just overwhelming
```suggestion
```